app/inputs/rich_text_input.rb

```diff
@@ -1,6 +1,5 @@
 class RichTextInput < SimpleForm::Inputs::TextInput
-  def input
-    input_html_options[:class] += [:wysihtml5] 
-    @builder.text_area(attribute_name, input_html_options)
+  def input(wrapper_options = nil)
```

Comment:
@joeyjoejoejr Yeah, this parameter was added in some newer version of Simpleform, so its use here is primarily to get rid of a deprecation warning. I'll add that underscore. Is that a Ruby convention?